configuration/driver/icedrv_step.F90

```diff
@@ -698,6 +700,110 @@ subroutine step_dyn_wave (dt)
 
       end subroutine step_dyn_wave
 
+
+!=======================================================================
+!
+! Run one time step of horizontal advection (if there is closing)
+!
+! authors: David Clemens-Sewall, NCAR
+
+      subroutine step_advection_scm (dt)
```

Comment:
See comments re nomenclature elsewhere

Reply:
Do you think it would make sense to rename `step_advection_scm` to `step_lateral_flux_scm`?

Reply:
Done.